Firmware/Sailwind/Linear_Guide/Linear_Guide.c

```diff
@@ -85,9 +98,18 @@ void Linear_Guide_move(Linear_Guide_t *lg_ptr, Loc_movement_t movement)
 	lg_ptr->localization.movement = movement;
 }
 
-void Linear_Guide_speed_ramp(Linear_Guide_t *lg_ptr)
+void Linear_Guide_set_desired_roll_trim_percentage(Linear_Guide_t *lg_ptr, int8_t percentage, LG_sail_adjustment_mode_t adjustment_mode)
```

Comment:
if adjustment_mode already contains the desired position of either roll or pitch then why is percentage an int8_t instead of uint8_t?